devenv/docker/blocks/loki/data/data.js

```diff
@@ -195,8 +196,31 @@ async function sendNewLogs() {
 }
 
 async function main() {
-  // we generate both old-logs and new-logs at the same time
-  await Promise.all([sendOldLogs(), sendNewLogs()])
+  function myFunction() {
```

Comment:
Is this test code? Why is it called `myFunction`?

Reply:
That happens when you blindly do a `git commit -a -m`... 🙈